include/boost/json/object.hpp

```diff
@@ -1567,6 +1614,12 @@ class object
     destroy(
         key_value_pair* first,
         key_value_pair* last) noexcept;
+
+    inline
```

Comment:
Could this use a little javadoc even though it's internal?

Reply:
eh.. 


by the way docca supports `@param firs,t last The range to destroy` now